spec/requests/reset_password_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+
+describe 'ResetPassword', type: :request do
+  let(:user) { create(:user) }
+
+  it 'get forgot password form' do
+    get '/forgot_password'
+    expect(response).to have_http_status(:success)
+  end
+
+  it 'get reset password form' do
+    get '/reset_password'
+    expect(response).to have_http_status(:success)
+  end
+
+  context 'right params when send letter' do
+    it 'return deliveries.size = 1' do
+      post '/auth/password', params: { email: user.email, redirect_url: 'http://localhost:3000/reset_password' }
+      expect(ActionMailer::Base.deliveries.size).to eq(1)
+    end
+  end
+
+  context 'incorrect data when send letter' do
+    it 'return error message' do
+      post '/auth/password', params: { email: user.email }
+      expect(ActionMailer::Base.deliveries.size).to eq(0)
+    end
+  end
+
+  context 'right params when get /auth/password/edit' do
+    it 'return found message' do
+      get '/auth/password/edit', params: { reset_password_token: user.send(:set_reset_password_token),
+                                           redirect_url: 'http://localhost:3000/reset_password' }
+      expect(response).to have_http_status(:found)
+    end
+  end
+
+  context 'incorrect data when get /auth/password/edit' do
+    it 'return error message' do
+      get '/auth/password/edit', params: { reset_password_token: user.send(:set_reset_password_token) }
+      expect(json_response['errors'].to_s).to include('Missing redirect URL')
+    end
+  end
+
+  context 'right params when reset password' do
+    it 'return success message' do
+      get '/auth/password/edit', params: { reset_password_token: user.send(:set_reset_password_token),
+                                           redirect_url: 'http://localhost:3000/reset_password' }
+      access_token = response.headers['location'].match(/access-token=(.+)&client=/)[1]
+      client = response.headers['location'].match(/client=(.+)&client_id=/)[1]
+      put '/auth/password', params: { password: user.password, password_confirmation: user.password,
```

Comment:
follow style and be consistent in what you do

Reply:
what are you meaning?

![image](https://user-images.githubusercontent.com/48195830/63297191-78175e80-c2d9-11e9-9cfa-ec90849c0c60.png)

Reply:
When I correct minor errors in the code, I write the name "Refactor code" and write a description in this commit, is this normal or no?
![image](https://user-images.githubusercontent.com/48195830/63297474-29b68f80-c2da-11e9-8caf-d75a515f78e7.png)

Reply:
be consistent means that if you make multiline hash - than all key/value pairs should be on separate lines.
About commits - the name Refactor code doesn't show what was done in commit. Commit name should answer on question Why the commit was done, and not how.